src/styles/card.scss

```diff
@@ -0,0 +1,86 @@
+.card {
+  width: 200px;
+  height: 406px;
+  box-sizing: border-box;
+
+  // border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+
+  // background-color: yellow;
+  // display: flex;
+  // justify-content: space-between;
+  // flex-direction: column;
+  // align-items: center;
+
+  &__logo {
+    width: 160px;
+    height: 134px;
+    background-image: url(../images/imac.jpeg);
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: contain;
+    margin-bottom: 40px;
+  }
+
+  &__description {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $text-color;
```

Comment:
The `color` property should use a variable for consistency and maintainability. Make sure that `$text-color` is defined in your variables file and represents the color intended for the text here.